src/models.ts

```diff
@@ -15,33 +21,18 @@ export type Release =
 
 export type ReleaseLike = Release
 
-export type ReleaseVersion = string
-
-// FIXME: generate proper types for processed release
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type ProcessedReleasesCollection = any
-
-export interface ProcessedReleaseChange extends ReleaseLike {
+export interface ProcessedRelease extends Omit<Release, 'body'> {
   title: string
   originalTitle: string
-  // Level: enumerate with error, warning, info or unknown
-  descriptionMdast: Parent
+  descriptionMdast: Root
 }
 
-// eslint-disable-next-line no-shadow
-export enum SemVerGroupTitles {
```

Comment:
[Enum types are not recommended](https://github.com/typescript-cheatsheets/react#enum-types), using union type of string literals instead.

Reply:
I like it, this is also easier to reason about for me.